group_project_v2/utils.py

```diff
@@ -338,3 +339,11 @@ def named_tuple_with_docstring(type_name, field_names, docstring, verbose=False,
 
     wrapper_class = type(type_name, (named_tuple_type,), {"__doc__": docstring})
     return wrapper_class
+
+
+def is_absolute(url):
+    """
+    :param url[str] url to asses
+    Returns a boolean value indicating if given `url` is absolute or not.
+    """
+    return bool(urlparse.urlparse(url).netloc)
```

Comment:
@ziafazal nit: might be good to add dedicated unit tests for this method. It is implicitly covered by the cases you've added to `test_send_request_no_data`, but having an explicit test for this method would still be helpful.